src/components/shared/MarkerTooltipContents.js

```diff
@@ -741,6 +749,21 @@ function getMarkerDetails(
               data.responseEnd,
               data.responseStart
             )}
+            {_markerDetailNullable(
```

Comment:
All of the request and response values are 0 on my test profile. Is there a correctness bug here?

<img width="227" alt="image" src="https://user-images.githubusercontent.com/1588648/49750561-40034a80-fc79-11e8-8c33-b05003294492.png">

https://deploy-preview-1547--perf-html.netlify.com/public/d18975390402fa97187fa8af7eac7f7a8569d3fa/network-chart/?globalTrackOrder=4-0-1-2-3&hiddenGlobalTracks=1-2&localTrackOrderByPid=18930-1-0~18933-0~&thread=4&v=3